airbyte-integrations/connectors/source-chargebee/source_chargebee/streams.py

```diff
@@ -287,3 +288,12 @@ class Event(IncrementalChargebeeStream):
     cursor_field = "occurred_at"
 
     api = EventModel
+
+class Coupon(IncrementalChargebeeStream):
+    """
+    API docs: https://apidocs.eu.chargebee.com/docs/api/coupon?prod_cat_ver=2#list_coupon
+    """
+
+    cursor_field = "occurred_at"
```

Comment:
`ocurred_at` is not in Coupon schema
```suggestion
    cursor_field = "updated_at"
```